### PR TITLE
test:supplement test cases for apis/opts/config/runtime.go

### PR DESCRIPTION
**Pull request by scy1993**
Signed-off-by: scy1993 <21851406@zju.edu.cn>

### Ⅰ. Describe what this PR did
test:supplement test cases for apis/opts/config/runtime.go

### CODE REVIEW

**Comment by chuanchang** [comment]
@scy1993 Thank you for your contribution, in fact, it exists a PR 2329 to add test cases and go through uncovered code path, please make sure it's not duplicate testing firstly.

**Comment by scy1993** [comment]
> @scy1993 Thank you for your contribution, in fact, it exists a PR #2329 to add test cases and go through uncovered code path, please make sure it's not duplicate testing firstly.

I just notice that you have already submitted the PR#2329, I will close my PR. Have a nice day.

File: apis/opts/config/runtime_test.go [hunk 1]
```diff
@@ -10,9 +10,11 @@ import (
 func TestNewRuntime(t *testing.T) {
 	assert := assert.New(t)
 
+	var tmp = map[string]types.Runtime(nil)
```
[the remark this answers is not in the document]
Comment (scy1993) [reply apis/opts/config/runtime_test.go]: Thank you for pointing out where I need to improve. I will pay attention to it in the future. Since there is a PR #2329 that has already add test cases, I will close my PR. Thanks for you advice anyway. Have a nice day.
